pom.xml

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>ir.sahab</groupId>
+  <artifactId>spring-boot-rest-commons</artifactId>
+  <name>Spring Boot Rest Commons</name>
+  <version>1.0.0</version>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <spring.boot.version>2.1.9.RELEASE</spring.boot.version>
+    <hibernate.core.version>5.3.12.Final</hibernate.core.version>
+    <logbook.version>2.0.0-RC.5</logbook.version>
+    <problem-spring-web.version>0.25.2</problem-spring-web.version>
```

Comment:
Remove it. It is just for one module. Write the version inline.

Reply:
Done.